portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,15 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetches comments from the servers and adds them to the DOM
+ */
+ function getComments(){
+    fetch('/data').then(response => response.json()).then((stats) => {
+        document.getElementById('comment1').innerText = stats[0];
+        document.getElementById('comment2').innerText = stats[1];
+        document.getElementById('comment3').innerText = stats[2];
```

Comment:
This looks great, and definitely works. But what happens if you want to add another comment? Let's try to refactor this to be more dynamic. For example: we can have one 'comment-list' in our html, loop through our comments here, and append each to the comment-list.

Here's an example (don't worry about the delete functionality in this example yet): https://github.com/googleinterns/step/blob/master/walkthroughs/week-3-server/examples/todo-list/src/main/webapp/script.js